### ISSUE TITLE
[Feature Request] resizing thumbnails

### ISSUE THREAD

**Issue by g07s** [yattee/yattee#453]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found feature request like this

### Type

New feature

### Describe the feature

Add resizing button for displaying thumbnails with 3 levels to choose: 
- small
- medium
- large

**Comment by bbsixzz** [comment]
Yes please, especially for tvOS it would be nice to fit 4 or 5 videos per row instead of 3.
